src/anim/binder/default-anim-binder.js

```diff
@@ -56,23 +56,38 @@ class DefaultAnimBinder {
                 var func = function (value) {
                     object.set.apply(object, value);
                 };
-                return new AnimTarget(func, 'vector', 3);
+                var targetIdentifier = AnimBinder.encode({
+                    entityPath: node.path,
+                    component: 'entity',
+                    propertyPath: 'localPosition'
+                });
+                return new AnimTarget(func, 'vector', 3, targetIdentifier);
```

Comment:
these get repeated bellow few times .. perhaps wrap it in a function?

Reply:
👍